engine/src/main/java/org/destinationsol/common/SolNullOptionalException.java

```diff
@@ -5,27 +5,19 @@
  * you may not use this file except in compliance with the License.
  * You may obtain a copy of the License at
  *
- * http://www.apache.org/licenses/LICENSE-2.0
+ *      http://www.apache.org/licenses/LICENSE-2.0
```

Comment:
Any reason for the indent?

Reply:
Screw you IDEA

Reply:
Actually, view upper. Bad template. Fixed.